libraries/testAutomation/testAutomation/test_runner.py

```diff
@@ -192,6 +202,16 @@ def copy_logs(self, destination: str = "./logs") -> None:
 
     def run(self, skip_delete: bool = False, keep_workflows: bool = False, developer_mode: bool = False,
             reuse_clusters: bool = False, test_reports_dir: str = None) -> None:
+        if developer_mode:
+            workflow_templates_dir = (
+                "../../TrafficCapture/dockerSolution/src/main/docker/migrationConsole/"
+                "workflows/templates/"
+            )
+            self.k8s_service.exec_migration_console_cmd([
+                "kubectl", "apply", "-f", workflow_templates_dir, "-n", "ma"
```

Comment:
We shouldn't need to run these commands on the migration console, check K8s service `run_command` function and look at is usages. Could also create a new function there that wraps this logic.